.github/workflows/browserslist-db-update.yml

```diff
@@ -0,0 +1,38 @@
+name: Update Browserslist Database
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
This makes sense, although is it necessary? Is this workflow subject to the default (restrictive) access? https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
Tested without it and it is not necessary. Removed. Thank you!